### ISSUE TITLE
Can't find  com.mpatric:mp3agic:jar:0.9.2-SNAPSHOT

### ISSUE THREAD

**Issue by ankushg** [pstonebu/overdrive#1]
Hi there, looks like the included `pom.xml` doesn't point to a repository that includes `com.mpatric:mp3agic:jar:0.9.2-SNAPSHOT`.

When I try to use the latest source from [the official `mp3agic` repository](https://github.com/mpatric/mp3agic), the `Id3v2Tag` class is missing the `getOverDriveMarkers()` method, which leads me to believe that the `0.9.2-SNAPSHOT` was actually a fork that included that additional method.

I had to cherry-pick this old commit into this repo in order to get things to build: https://github.com/pstonebu/overdrive/commit/268af01a4d361f789c053c57a044625552dc038e

---

Would it be possible to publish the source for the forked version of `mp3agic` which includes the `getOverDriveMarkers()` method?

**Comment by pstonebu** [comment]
Yeah, I ran into issues getting the necessary changes into mp3agic. Attaching the patch to mp3agic that I use locally.
[overdrive_markers.txt](https://github.com/pstonebu/overdrive/files/4611968/overdrive_markers.txt)
